lib/kovid/constants.rb

```diff
@@ -123,5 +123,17 @@ module Constants
       compare_us_states: [1, 2, 3, 4, 5],
       compare_provinces: [1, 2, 3]
     }.freeze
+
+    COUNTRY_CASES_DEATHS = [
```

Comment:
Can include other information in this table so it looks similar to this:

```
+--------+-------------+--------+--------------+-----------+---------+
|                             🇵🇱​ POLAND                              |
+--------+-------------+--------+--------------+-----------+---------+
| Cases  | Cases Today | Deaths | Deaths Today | Recovered | Tests   |
+--------+-------------+--------+--------------+-----------+---------+
| 17,062 | +141        | 847    | +8           | 9,805     | 507,778 |
+--------+-------------+--------+--------------+-----------+---------+
```
Same with `STATE_CASES_DEATHS` :)

Reply:
I'm talking about the headings(kind of info we want)*

Reply:
It seems nice to have `--full` options too.

Reply:
Adding `--full` will be too many commands/work. You can just fetch the full info for the table by default so there'll be no need for `--full`. Users can then just look and find what they want.

Reply:
Nice.